### PR TITLE
feat: Table global css

### PR DESCRIPTION
**Pull request by flecmen**

### CODE REVIEW

File: Breadcrumbs/Breadcrumbs.vue [hunk 1]
```diff
@@ -74,7 +74,7 @@ function getLabel(breadcrumb: IBreadcrumb) {
   --apply: flex grow flex-gap-x-1 items-center text-sm m-t-2 m-b-1 overflow-auto;
 
   &-wrapper {
-    --apply: flex flex-gap-x-1 items-center md:p-x-3;
+    --apply: flex flex-gap-x-1 items-center md:p-$breadcrumbs-padding;
```
Comment (Kylyi) [inline Breadcrumbs/Breadcrumbs.vue:77]: `B`

File: Table/TableRow.vue [hunk 2]
```diff
@@ -98,7 +98,7 @@ const isEditable = computedEager(() => {
 
 .tr:hover {
   .cell {
-    --apply: bg-blue-500/10;
+    --apply: bg-$Table-row-hover-bg;
```
Comment (Kylyi) [inline Table/TableRow.vue:101]: its weird to call it `row-hover` if it's on `cell`...

File: Table/models/table-column.model.ts [hunk 3]
```diff
@@ -324,6 +324,7 @@ export class TableColumn<T = IItem> implements IItemBase<T> {
       case 'date':
       case 'timestamp':
       case 'duration':
+      case 'stringEnum':
```
Comment (Kylyi) [inline Table/models/table-column.model.ts:327]: Shouldn't be part of the `components`

File: Table/models/table-column.model.ts [hunk 4]
```diff
@@ -517,8 +518,8 @@ export class TableColumn<T = IItem> implements IItemBase<T> {
     // Filtering
     this.filters = col.filters ? col.filters : []
     this.filterComponent = col.filterComponent
-    this.comparators = col.comparators
-    this.comparator = col.comparator || this.comparator
+    this.comparators = col.comparators ?? config.dataTypes.comparators[this.dataType]
+    this.comparator = col.comparator ?? config.dataTypes.defaultComparator[this.dataType] ?? this.comparator
```
Comment (Kylyi) [inline Table/models/table-column.model.ts:522]: the `defaultComparator` needs to be added to the `components-config.ts` also

Comment (Kylyi) [reply Table/models/table-column.model.ts:522]: im not sure in general about this... we alredy have the `COMPARATORS_BY_DATATYPE_MAP` that is used for resolving the comparators...

File: Tabs/Tabs.vue [hunk 5]
```diff
@@ -138,6 +138,7 @@ watch(model, model => {
             ...(tab.id === activeTab?.id ? [labelActiveClass] : []),
             { 'is-active': tab.id === activeTab?.id },
           ]"
+          v-bind=tabBtnProps
```
Comment (Kylyi) [inline Tabs/Tabs.vue:141]: where are `"`? :)

File: Table/models/table-column.model.ts (hunk 3 above)
Comment (flecmen) [reply Table/models/table-column.model.ts:327]: Leaving after dm discussion - there is no good alternative present.

**Pull request merged**